aries-site/src/examples/templates/wizard/WizardValidationExample.js

```diff
@@ -0,0 +1,346 @@
+  active: PropTypes.number.isRequired,
+  formValues: PropTypes.shape({}).isRequired,
+  setActive: PropTypes.func.isRequired,
+};
+
+const StepTwo = ({ active, setActive }) => (
+  <>
+    <Box margin={{ bottom: 'large' }}>
+      <FormField label="Select" htmlFor="select" name="select">
+        <Select
+          placeholder="Select item"
+          id="select"
+          name="select"
+          options={['Option 1', 'Option 2']}
+        />
+      </FormField>
+      <FormField htmlFor="checkboxgroup" label="Label" name="checkboxgroup">
+        <CheckBoxGroup
+          id="checkboxgroup"
+          name="checkboxgroup"
+          options={['CheckBox 1', 'CheckBox 2']}
+        />
+      </FormField>
+      <FormField
+        help="Description of how to use this field"
+        htmlFor="text-area"
+        label="Label"
+        name="text-area"
+      >
+        <TextArea
+          id="text-area"
+          name="text-area"
+          options={['CheckBox 1', 'CheckBox 2']}
+          placeholder="Placeholder text"
+        />
+      </FormField>
+    </Box>
+    <Button
+      fill="horizontal"
+      label="Next"
+      icon={<FormNextLink />}
+      primary
+      reverse
+      onClick={() => setActive(active + 1)}
+    />
+  </>
+);
+
+StepTwo.propTypes = {
+  active: PropTypes.number.isRequired,
+  setActive: PropTypes.func.isRequired,
+};
+
+const data = [
+  'Summary value of step 1',
+  'More summary value of step 1',
+  'Summary value of step 2',
+  'More summary values from step 2',
+];
+
+const StepThree = () => {
+  return (
+    <>
+      <Box gap="small" margin={{ bottom: 'large' }}>
+        <List data={data} pad={{ horizontal: 'none', vertical: 'small' }}>
+          {(datum, index) => (
+            <Box key={index} direction="row" gap="medium" align="center">
+              <Checkmark color="text-strong" size="small" />
+              <Text color="text-strong">{datum}</Text>
+            </Box>
+          )}
+        </List>
+        <Text color="text-strong">
+          Include guidance to what will occur when “Finish Setup” is clicked.
+        </Text>
+      </Box>
+      <Button fill="horizontal" label="Finish Setup" primary type="submit" />
+    </>
+  );
+};
+
+const steps = [
+  {
+    description: `Step one description. Keep each step simple and in chunks 
+    easy enough to fit on a single page.`,
+    inputs: (active, setActive, formValues) => (
+      <StepOne active={active} setActive={setActive} formValues={formValues} />
+    ),
+  },
+  {
+    description: 'Step 2 description.',
+    inputs: (active, setActive) => (
+      <StepTwo active={active} setActive={setActive} />
+    ),
+  },
+  {
+    description: 'Provide a summary of what was accomplished or configured. ',
+    inputs: (active, setActive) => (
+      <StepThree active={active} setActive={setActive} />
+    ),
+    title: 'Finish title',
+  },
+];
+
+export const WizardValidationExample = () => {
+  const [active, setActive] = useState(1);
```

Comment:
Soft opinion here.

Would `activeStep` or `activeIndex` help for clarity?

Reply:
I like `activeStep` let me make that update for this file here. I will update the other wizard files in a separate PR.

Reply:
For readability, I have changed `active` to `activeIndex` with default state of 0. This will be more aligned with typical array conventions/expectations. However, I've also introduced `activeStep` which is used for displaying the numeric value of the step in the UI. This is automatically updated by a useEffect whenever activeIndex updates.